ruby/examples/lib/register_user_with_form.rb

```diff
@@ -0,0 +1,39 @@
+require 'active_model'
+
+class RegisterUserWithForm
```

Comment:
See the, "how do future developers (of new features) know to find the gateway inside another use case" argument